packages/services/package.json

```diff
@@ -22,6 +23,10 @@
     "@babel/core": "^7.7.7",
     "@babel/node": "^7.7.7",
     "@babel/preset-env": "^7.7.7",
-    "nodemon": "^2.0.2"
+    "chai": "^4.2.0",
+    "mocha": "^7.1.2",
+    "nodemon": "^2.0.2",
+    "sinon": "^9.0.2",
```

Comment:
be really careful about this library, maybe in a few years you will understand me :)